package.json

```diff
@@ -3,10 +3,12 @@
     "@wireapp/antiscroll-2": "1.0.2",
     "@wireapp/avs": "5.1.40",
     "@wireapp/commons": "2.2.23",
-    "@wireapp/core": "14.4.2",
+    "@wireapp/core": "15.1.1",
     "@wireapp/protocol-messaging": "1.24.2",
     "@wireapp/react-ui-kit": "7.19.2",
-    "@wireapp/store-engine-dexie": "0.7.0",
+    "@wireapp/store-engine": "4.1.3",
```

Comment:
I think we can remove this dependency because it's included in the core already

Reply:
I thought so too but "@wireapp/store-engine-dexie" reported with a warning that it is missing a peer dependency.